### ISSUE TITLE
If ``$HOME`` is not set on linux, show an error to the user.

### ISSUE THREAD

**Issue by ebfull** [zcash/zcash#629]
See https://github.com/Electric-Coin-Company/zcash/commit/4f1c37980e13167f01cd6fdc07b201bf4dabbf52#commitcomment-15544153

**Comment by adamhoracek** [comment]
On OSX it says this...

```
Adams-MacBook-Pro:zcash adamλ ./src/zcashd


************************
EXCEPTION: St13runtime_error
Could not open proving key file: /Users/user/Library/Application Support/ZcashParams/zc-testnet-public-alpha-proving.key
Please refer to user documentation for installing this file.
bitcoin in AppInit()
```

**Comment by ebfull** [comment]
This will be addressed by #1312.
